### PR TITLE
load-aware: make trimaran log more visible

### PR DESCRIPTION
**Pull request by drewrip**
Always dump trimaran log after scale test and link to it in the report.

### CODE REVIEW

File: visualizations/load-aware/store/parsers.py [hunk 1]
```diff
@@ -233,11 +234,24 @@ def _parse_nodes_info(dirname, sutest_cluster=True):
...
+    with open(register_important_file(dirname, filename)) as f:
         sutest_ocp_version_yaml = yaml.safe_load(f)
 
     return sutest_ocp_version_yaml["openshiftVersion"]
 
+def _parse_file_locations(dirname):
+    file_locations = types.SimpleNamespace()
+
+    file_locations.trimaran_log = artifact_paths.LOAD_AWARE_SCALE_TEST_DIR / "trimaran_scheduler.log"
+    register_important_file(dirname, file_locations.trimaran_log)
```
Comment (kpouget) [inline visualizations/load-aware/store/parsers.py:248]: you need to add `trimaran_scheduler.log` in the list of important at the beginning of `parsers.py`
(`register_important_file` is a bit of a misnomer, it's more checking that the file is part of the list IIRC)

File: visualizations/load-aware/plotting/pod_times.py [hunk 2]
```diff
@@ -168,7 +179,38 @@ def do_plot(self, ordered_vars, settings, setting_lists, variables, cfg):
         title = f"Pod execution timeline"
 
         fig.update_layout(title=title, title_x=0.5)
-
+       
+        scheduling_phase = df.where(df["Phase"] == "Scheduling")
+        q1, med, q3 = stats.quantiles(scheduling_phase.Duration)
+        sched_min = scheduling_phase.min()
+        sched_max = scheduling_phase.max()
```
Comment (kpouget) [inline visualizations/load-aware/plotting/pod_times.py:186]:
```suggestion
        sched_min = scheduling_phase["Duration"].min()
        sched_max = scheduling_phase["Duration"].max()
```

File: visualizations/load-aware/plotting/pod_times.py [hunk 3]
```diff
@@ -168,7 +179,38 @@ def do_plot(self, ordered_vars, settings, setting_lists, variables, cfg):
         title = f"Pod execution timeline"
 
         fig.update_layout(title=title, title_x=0.5)
-
+       
+        scheduling_phase = df.where(df["Phase"] == "Scheduling")
```
Comment (kpouget) [inline visualizations/load-aware/plotting/pod_times.py:183]:
```suggestion
        scheduling_phase = df[df.Phase == "Scheduling"]
```
wasn't straightforward to troubleshoot :D

Comment (kpouget) [reply visualizations/load-aware/plotting/pod_times.py:183]:
```
                   Start                 End            Creation  Duration                    Pod                                        Node Workload       Phase
0    2023-07-24 16:39:41 2023-07-24 16:39:41 2023-07-24 16:39:41       0.0  coreutils-make-pod-n0  ip-10-0-142-236.us-west-2.compute.internal     make  Scheduling
1                    NaT                 NaT                 NaT       NaN                    NaN                                         NaN      NaN         NaN
2                    NaT                 NaT                 NaT       NaN                    NaN                                         NaN      NaN         NaN
601                  NaT                 NaT                 NaT       NaN                    NaN                                         NaN      NaN         NaN
```

**Comment by drewrip** [comment]
[test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/860/pull-ci-openshift-psap-ci-artifacts-main-load-aware-light/1683769341566259200/artifacts/light/test/artifacts/001__default_plots/_matbench_visualize.log). Parsing is running ok.

@kpouget should be good to merge

**Comment by kpouget** [comment]
thanks @drewrip , looks good :) 
merging.

**Pull request merged**